### PR TITLE
feat(DescriptionList): apply tokens

### LINKED ISSUE #6101: Description list tokens

### PR DESCRIPTION
**Pull request by kmcfaul**
closes #6101 

with new v6 branch. Old PR: #6102

### CODE REVIEW

File: src/patternfly/components/DescriptionList/description-list.scss [hunk 1]
```diff
@@ -54,26 +53,27 @@ $pf-v5-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "m
...
   --#{$description-list}__text--m-help-text--TextDecorationOffset: #{pf-size-prem(4px)};
-  --#{$description-list}__text--m-help-text--hover--TextDecorationColor: var(--#{$pf-global}--Color--100);
-  --#{$description-list}__text--m-help-text--focus--TextDecorationColor: var(--#{$pf-global}--Color--100);
+  --#{$description-list}__text--m-help-text--hover--TextDecorationColor: var(--pf-t--global--border--color--hover);
+  --#{$description-list}__text--m-help-text--focus--TextDecorationColor: var(--pf-t--global--border--color--hover);
 
   // Display modifiers
-  --#{$description-list}--m-display-lg__description--FontSize: var(--#{$pf-global}--FontSize--lg);
-  --#{$description-list}--m-display-2xl__description--FontSize: var(--#{$pf-global}--FontSize--2xl);
+  --#{$description-list}--m-display-lg__description--FontSize: var(--pf-t--global--font--size--body--lg); // TODO replace with new font tokens when available
```
Comment (thatblindgeye) [inline src/patternfly/components/DescriptionList/description-list.scss:63]: Looks like this is setting the font size to just 16px, Figma show it shouls be 22px for the description text for large styling.

File: src/patternfly/components/DescriptionList/description-list.scss [hunk 2]
```diff
@@ -54,26 +53,27 @@ $pf-v5-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "m
...
+  --#{$description-list}--m-display-2xl__description--FontSize: var(--pf-t--global--font--size--heading--lg); // TODO replace with new font tokens when available
```
Comment (thatblindgeye) [inline src/patternfly/components/DescriptionList/description-list.scss:64]: Figma only has sizing for small, default, and large for the card variant of the component. Do we still need this var and the associated modifier class?

**Review by thatblindgeye — COMMENTED** [review]
Noticing a couple of things:

- This is pre-existing so we can do a followup, but the "Help text" example should have teh `pf-m-help-text` class applied to the `.pf-v5-c-description-list__term > .pf-v5-c-description-list__text` element to apply the underlined styling
- Figma has different font size styling for card variants than shown here. There's a "small" variant that looks to use the font sizes of a default description list, a "default" variant that has slightly larger font sizing, and a "large" variant. The `__term` element also has its font size updated (right now we're only updating the `__description` font sizing).
- The text blends in with the card variant in dark mode, but I imagine that would be resolve when the card component gets Penta-fied.

File: src/patternfly/components/DescriptionList/description-list.scss (hunk 2 above)
Comment (kmcfaul) [reply src/patternfly/components/DescriptionList/description-list.scss:64]: @mcoker what do you think?

File: src/patternfly/components/DescriptionList/description-list.scss [hunk 3]
```diff
@@ -54,26 +53,27 @@ $pf-v5-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "m
   --#{$description-list}--m-auto-fit--GridTemplateColumns--minmax--min: var(--#{$description-list}--m-auto-fit--GridTemplateColumns--min);
 
   // help text
-  --#{$description-list}__text--m-help-text--TextDecorationColor: var(--#{$pf-global}--BorderColor--200);
-  --#{$description-list}__text--m-help-text--TextDecorationThickness: var(--#{$pf-global}--BorderWidth--sm);
+  --#{$description-list}__text--m-help-text--TextDecorationColor: var(--pf-t--global--icon--color--regular);
+  --#{$description-list}__text--m-help-text--TextDecorationThickness: var(--pf-t--global--border--width--100);
```
Comment (mcoker) [inline src/patternfly/components/DescriptionList/description-list.scss:57]: Just a note here, but we'll want to update this to use a semantic token

**Review by mcoker — APPROVED** [review]
L🎄TM! Left a couple of comments we can follow up on.

**Review by lboehling — APPROVED** [review]
lgtm! thank you!

File: src/patternfly/components/DescriptionList/description-list.scss [hunk 4]
```diff
@@ -54,26 +53,27 @@ $pf-v5-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "m
   --#{$description-list}--m-auto-fit--GridTemplateColumns--minmax--min: var(--#{$description-list}--m-auto-fit--GridTemplateColumns--min);
 
   // help text
-  --#{$description-list}__text--m-help-text--TextDecorationColor: var(--#{$pf-global}--BorderColor--200);
-  --#{$description-list}__text--m-help-text--TextDecorationThickness: var(--#{$pf-global}--BorderWidth--sm);
+  --#{$description-list}__text--m-help-text--TextDecorationColor: var(--pf-t--global--icon--color--regular);
```
[the remark this answers is not in the document]
Comment (kmcfaul) [reply src/patternfly/components/DescriptionList/description-list.scss:56]: @mcoker Opened #6223 to add the suggestions

**Pull request merged**